src/main/java/me/karakelley/http/handlers/QueryParametersHandler.java

```diff
@@ -0,0 +1,50 @@
+package me.karakelley.http.handlers;
+
+import me.karakelley.http.Request;
+import me.karakelley.http.Response;
+import me.karakelley.http.UriQueryParser;
+import me.karakelley.http.contentpresenter.ContentPresenter;
+import me.karakelley.http.exceptions.InvalidRequestException;
+import me.karakelley.http.responses.BadRequest;
+import me.karakelley.http.responses.Ok;
+
+import java.util.*;
+
+public class QueryParametersHandler implements Handler {
+  private final String CONTENT_TYPE = "Content-Type";
+  private final String TEXT_HTML = "text/html";
+  private ContentPresenter contentPresenter;
+
+  public QueryParametersHandler(ContentPresenter contentPresenter) {
+    this.contentPresenter = contentPresenter;
+  }
+
+  @Override
+  public Response respond(Request request) {
+    Response response = new Ok();
+    try {
+      return serveQueryResponse(response, request);
+    } catch (InvalidRequestException e) {
+      return new BadRequest();
+    }
+  }
+
+  private Response serveQueryResponse(Response response, Request request) {
+    Map<String, List<String>> query = UriQueryParser.parse(request.getPath());
+    if (!query.isEmpty()) {
+      response.setBody(contentPresenter.displayResources(createDescriptionList(query)));
```

Comment:
Using `ContentPresenter` here feels wrong. It feels like what you need here and what you need in static files handler are likely to change for completely different reasons. My gut says that `ContentPresenter` is too vague of a name and it leads one to do this.